src/Symfony/Bridge/Monolog/Handler/DebugHandler.php

```diff
@@ -22,6 +22,29 @@
  */
 class DebugHandler extends TestHandler implements DebugLoggerInterface
 {
+    private $skipOnCli = true;
+
+    public function __construct($level = Logger::DEBUG, $bubble = true)
+    {
+        parent::__construct($level, $bubble);
+
+        if (func_num_args() >= 3) {
+            $this->skipOnCli = (bool) func_get_arg(2);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isHandling(array $record)
+    {
+        if ($this->skipOnCli && php_sapi_name() === 'cli') {
```

Comment:
The `php_sapi_name() === 'cli'` can also be done in the constructor with the `skipOnCli` property.